package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "consus-client",
   "description": "The client application for Consus, an inventory management system",
-  "version": "0.1.0",
+  "version": "0.2.0",
```

Comment:
Not worthy of not approving, but should the version number really be changed in this PR?